internal/cluster/transition_test.go

```diff
@@ -1504,6 +1573,9 @@ var _ = Describe("Refresh Cluster - With DHCP", func() {
 				} else {
 					cluster.MachineNetworkCidrUpdatedAt = time.Now().Add(-3 * time.Minute)
 				}
+				if t.srcState == models.ClusterStatusInstalled {
+					cluster.ValidationsInfo = getSuccessfulValidationInfo()
```

Comment:
Why do you need it? it's not clear why do you overwrite the validations info?

Reply:
I added a comment